src/main/java/Duke.java

```diff
@@ -6,5 +9,68 @@ public static void main(String[] args) {
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?\n");
+        ArrayList<Task> tasks = new ArrayList<Task>();
+        try (Scanner scan = new Scanner(System.in)) {
+            String input = scan.nextLine();
+            String output = new String();
+            while (!input.equals("bye")) {
+                if (input.equals("list")) {
+                    output = "Here are the tasks in the list:" + System.lineSeparator() + Task.printTasksList(tasks);
+
+                } else if (input.matches("mark [0-9]{1,2}")) {
+                    String[] marks = input.split(" ");
+                    tasks.get(Integer.parseInt(marks[1]) - 1).markAsDone();
+                    output = "Nice! I've marked this task as done:" + System.lineSeparator()
+                            + tasks.get(Integer.parseInt(marks[1]) - 1).toString();
+                } else if (input.matches("unmark [0-9]{1,2}")) {
+                    String[] marks = input.split(" ");
+                    tasks.get(Integer.parseInt(marks[1]) - 1).markAsNotDone();
+                    output = "OK, I've marked this task as not done yet:" + System.lineSeparator()
+                            + tasks.get(Integer.parseInt(marks[1]) - 1).toString();
+                } else if (input.startsWith("todo", 0)) {
+                    String toDoDesc = input.split("todo")[1].trim();
+                    Task toDo = new Todo(toDoDesc);
+                    tasks.add(toDo);
+                    output = "Got it. I've added this task:" + System.lineSeparator()
+                            + toDo.toString() + System.lineSeparator() + "Now you have " + Task.numberOfTasks
+                            + " in the list.";
+                } else if (input.startsWith("deadline", 0)) {
+                    String deadlineDesc = input.split("/")[0].split("deadline")[1].trim();
+                    String deadlineDay = input.split("/")[1].trim();
+                    Task deadline = new Deadline(deadlineDesc, deadlineDay);
+                    tasks.add(deadline);
+                    output = "Got it. I've added this task:" + System.lineSeparator()
+                            + deadline.toString() + System.lineSeparator() + "Now you have " + Task.numberOfTasks
+                            + " in the list.";
+                } else if (input.startsWith("event", 0)) {
+                    String eventDesc = input.split("/")[0].split("event")[1].trim();
+                    String start = input.split("/")[1].trim();
+                    String end = input.split("/")[2].trim();
+                    Task event = new Event(eventDesc, start, end);
+                    tasks.add(event);
+                    output = "Got it. I've added this task:" + System.lineSeparator()
+                            + event.toString() + System.lineSeparator() + "Now you have " + Task.numberOfTasks
```

Comment:
A lot of "magic numbers" involved. Maybe can consider using constants instead?

Reply:
Try not to use magic numbers. Can consider using an aptly named variable.

Reply:
Can consider using switch case instead of if else to decide what to do based on the first word entered